google-cloud-datastore/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.togglz</groupId>
+        <artifactId>togglz-project</artifactId>
+        <version>2.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>togglz-google-cloud-datastore</artifactId>
+    <name>Togglz - Google Cloud Datastore Integration</name>
+    <description>Togglz - Google Cloud Datastore Integration</description>
+
+    <properties>
+        <gcloud.version>0.8.0-beta</gcloud.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.togglz</groupId>
+            <artifactId>togglz-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-datastore</artifactId>
+            <version>${gcloud.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Hi @chkal ,

Just checked here and guava is a transitive dependency.
We cant get rid of it.
Sorry..